### PR TITLE
Add loading indicator so it's clear when a request is being made

### PR DESCRIPTION
**Pull request by paulstatezny**
### Acceptance Criteria
1. While a request is being made, there is some visual indicator that it's being made. (That your button click was registered.)

### Tasks
- Add indicator -- just text for now. Can be cleaned up when Lively is redesigned.

### CODE REVIEW

**Comment by paulstatezny** [comment]
:star:

**Comment by bobeagan** [comment]
:interrobang: The .DS_Store is something that you should block at your global gitignore level.  I don't think it belongs in the repo's gitignore.

**Comment by paulstatezny** [comment]
:shrug: ok

**Comment by bobeagan** [comment]
Guessing you meant to send this back into review :star:

File: application/ui/components/button.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    propTypes : {
+        className : React.PropTypes.string,
+        ref       : React.PropTypes.string,
+        onClick   : React.PropTypes.func
+    },
+
+    onClick : function(event)
+    {
+        if (this.props.disabled === true) {
```
Comment (agerbens) [inline application/ui/components/button.jsx]: Should `this.props.disabled` be listed in propTypes?

**Comment by agerbens** [comment]
:interrobang:

**Comment by paulstatezny** [comment]
:star:

**Comment by agerbens** [comment]
:pray:

**Comment by pcavagna** [comment]
:hand:

**Comment by aecarr2** [comment]
:hand:

**Comment by aecarr2** [comment]
:+1:

**Pull request merged**